packages/client-core/src/admin/components/Instance/InstanceTable.tsx

```diff
@@ -87,12 +95,17 @@ const InstanceTable = (props: Props) => {
       locationId: locationId?.name || '',
       channelId,
       action: (
-        <>
-          <a href="#h" className={classes.actionStyle}>
-            {' '}
-            <span className={classes.spanDange}>Delete</span>{' '}
-          </a>
-        </>
+        <a
+          href="#h"
+          className={classes.actionStyle}
+          onClick={() => {
+            setPopConfirmOpen(true)
+            setInstanceId(id)
```

Comment:
getting error on deleting an instance

<img width="451" alt="Screenshot 2022-01-13 at 4 48 31 PM" src="https://user-images.githubusercontent.com/30355034/149325004-dc6b152f-0486-4ab6-bdc2-26b51b78336a.png">

Reply:
I am not able to get this error. which step are you following to delete instance ?

Reply:
simply pressed delete and confirmed from popup and that instance had 0 users.

Reply:
I tried to test it, but I am not able to get an error. could you please test again with latest changes

Reply:
- open two instance of location/test in separate browsers
- then go to /admin/instance route
- You will see instances like in image#1
- delete instance with 3031 port image#2
- confirm the deletion and the screen will go blank like in image#3

image#1
<img width="1216" alt="Screenshot 2022-01-13 at 8 55 40 PM" src="https://user-images.githubusercontent.com/30355034/149364163-5530b6c2-e6b2-4d10-8454-6e58e92df7ec.png">

image#2
<img width="1216" alt="Screenshot 2022-01-13 at 8 55 57 PM" src="https://user-images.githubusercontent.com/30355034/149364189-b9a58eb2-e964-4c7e-9fd7-b1c702324e9e.png">

image#3
<img width="1673" alt="Screenshot 2022-01-13 at 8 56 09 PM" src="https://user-images.githubusercontent.com/30355034/149364201-c1b84730-ae78-4e6d-aa2c-5a0c089f0f4a.png">